app/src/main/res/layout/item_tx_queued_contract_interaction.xml

```diff
@@ -61,7 +60,7 @@
         android:gravity="end"
         android:lineSpacingExtra="4sp"
         android:text="@string/tx_list_change_mastercopy"
-        android:maxLines="2"
+        android:singleLine="true"
```

Comment:
This will lead to commands being cut off like this:
![image](https://user-images.githubusercontent.com/246473/104172936-530f1880-5405-11eb-9a2d-c5290650d151.png)
@posthnikova What do you prefer ellipsized commands or commands wrongly broken into two lines?